grails-app/conf/UrlMappings.groovy

```diff
@@ -342,6 +342,16 @@ class UrlMappings {
             action = [DELETE: "removeItem"]
         }
 
+        "/api/replenishments/$id/createPicklist"(parseRequest: true) {
```

Comment:
I'd like to avoid remote procedure call naming convention so let's go with a subresource. I can't tell if we want the subresource should be picklist or picklistItems. 
```
"/api/replenishments/$id/picklists"(parseRequest: true) {
    controller = { "replenishmentApi" }
    action = [POST: "createPicklist", PUT: "updatePicklist", DELETE: "deletePicklist"]
}
```

Reply:
My guess would be picklistItems as the subresource and methods could be createPicklistItems, updatePicklistItems, deletePicklistItems.